hooks/useHydrateStore.tsx

```diff
@@ -17,8 +17,8 @@ export default function useHydrateStore() {
   } = useWalletStore((s) => s.actions)
   useEffect(() => {
     const fetch = async () => {
-      const realmInfo = getRealmInfo(symbol as string, apiEndpoint)
       setConnectionConfig(apiEndpoint)
+      const realmInfo = getRealmInfo(symbol as string, apiEndpoint)
```

Comment:
just moved this since it related to the code below

Reply:
:+1: